app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -374,15 +374,12 @@ protected void onSaveInstanceState(Bundle outState) {
     @Override
     protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         super.onActivityResult(requestCode, resultCode, data);
-
         String result = null;
         switch (requestCode) {
             case BARCODE_CAPTURE:
                 if (resultCode == Activity.RESULT_OK) {
-                    result = data.getStringExtra("SCAN_RESULT");
-                    String dbg = "Got url from barcode scanner: " + result;

```

Comment:
This output is actually _quite_ helpful sometimes and I'd like to keep it in,

Reply:
Good call